### PR TITLE
Store the root certificates as a multi-line string literal.

### PR DESCRIPTION
**Pull request by MrMage**
This makes reviews easier and avoids expensive memory allocations and Base64 conversions at runtime.

The signature of `roots_pem` changes from `String`-returning function to `String` constant; however, that method has only been made public a few days ago, so no users should be affected.

### CODE REVIEW

**Comment by MrMage** [comment]
My hunch is that CI is starting to get flaky again (no idea about the reasons, that should probably be investigated soon); closing and re-opening to re-trigger CI.

File: Sources/RootsEncoder/main.swift [hunk 1]
```diff
@@ -98,13 +98,9 @@ if CommandLine.arguments.contains("test") {
   s += "import Foundation\n"
   s += "\n"
 
-  s += "public func roots_pem() -> String {\n"
+  s += "public let roots_pem: String = \"\"\"\n"
```
Comment (rebello95) [inline Sources/RootsEncoder/main.swift]: Should we make this a constant using the `k` prefix (i.e., `kRootsPem`)?

Comment (MrMage) [reply Sources/RootsEncoder/main.swift]: Great idea! I've renamed the constant to `kRootCertificates` now; what do you think of that name?

Comment (rebello95) [reply Sources/RootsEncoder/main.swift]: Yea sounds great!

**Pull request merged**